### PR TITLE
update c2c sample to include JS bundle

### PR DESCRIPTION
**Pull request by dmceachernmsft**
## Purpose

* ...
Includes the JS bundle in the completed react code as a avenue to test the c2c experience
## Does this introduce a breaking change?

```
[ ] Yes
[ x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: ui-library-click-to-call/README.md [hunk 1]
```diff
@@ -1,23 +1,46 @@
...
+To setup the completed `react` project, In the `ui-library-click-to-call` directory, run:
 
 ### `npm install`
 
 ### Swap placeholders for identifiers
-Go to the `src` folder and find the `App.tsx` file.
-There input values for the `token`, `userId` and `locator`.
+
+After the install is done go to the `src` folder and find the `App.tsx` file.
+There input values for the `token`, `userId` and `teamsAppIdentifier`.
```
Comment (JamesBurnside) [inline ui-library-click-to-call/README.md:20]:
```suggestion
There are input values for the `token`, `userId` and `teamsAppIdentifier`.
```

**Review by JamesBurnside — APPROVED** [review]

File: ui-library-click-to-call/README.md [hunk 2]
```diff
@@ -46,12 +64,47 @@ Your app is ready to be deployed!
...
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+## Cross platform Javascript bundle
+
+This project also includes a bundle that can be used to run the `react` based `CallComposite` pre-configured for use as a click
```
Comment (mgamis-msft) [inline ui-library-click-to-call/README.md:69]: super nit: I would edit to '... run the `React` based `CallComposite`,  pre-configured for use ...'

File: ui-library-click-to-call/README.md [hunk 3]
```diff
@@ -46,12 +64,47 @@ Your app is ready to be deployed!
...
+to call experience. This bundle and a sample `index.html` file can be found in the `cross-platform-js` folder and can be used by other frameworks by importing it through a script tag.
+
+```html
+<script src="./outboundCallComposite.js"></script>
+```
+
+### Setting up the bundle
+
+In the `index.html` file find the `userId`, `token`, and `displayName` values and enter yor information like in the `react` app.
```
Comment (mgamis-msft) [inline ui-library-click-to-call/README.md:78]: Should react be capitalized?

File: ui-library-click-to-call/README.md [hunk 4]
```diff
@@ -46,12 +64,47 @@ Your app is ready to be deployed!
...
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+## Cross platform Javascript bundle
```
Comment (mgamis-msft) [inline ui-library-click-to-call/README.md:67]: super nit: Java**S**cript

**Review by mgamis-msft — APPROVED** [review]

**Pull request merged**